iati/about/models.py

```diff
@@ -9,7 +11,7 @@
 from home.models import AbstractContentPage, AbstractIndexPage, DefaultPageHeaderImageMixin, PullQuoteBlock
 
 
-class AboutPage(DefaultPageHeaderImageMixin, AbstractContentPage):
+class AboutPage(DefaultPageHeaderImageMixin, AbstractContentPage):  # pylint: disable=too-many-ancestors
```

Comment:
As this ancestors linter fix is so prolific, it's probably best to specify it at a higher level in `setup.cfg`

Reply:
Kk – reverted f912bbd and fixed in 2ac2b11.